appender_test.go

```diff
@@ -411,7 +411,7 @@ func TestAppenderFlushMetric(t *testing.T) {
 			if !ignoreCount {
 				assert.Equal(t, count, int(dp.Count))
 			} else {
-				assert.Greater(t, int(dp.Count), count)
+				assert.GreaterOrEqual(t, int(dp.Count), count)
```

Comment:
I debugged it locally with the main branch, and it also quite often emits 2 counts here, it should be ok to expect greater or equal than 2 if I'm not mistaken.